on-chain/contracts/User.sol

```diff
@@ -48,16 +48,16 @@ contract User is Ownable, TAParser {
         uint success;
         (success, tokens, number) = JsmnSolLib.parse(out, 50);
 
-        uint isErrIdx = 2;
-        string memory isErr = JsmnSolLib.getBytes(out, tokens[isErrIdx].start, tokens[isErrIdx].end);
+        uint successIdx = 4;
+        string memory resultSuccess = JsmnSolLib.getBytes(out, tokens[successIdx].start, tokens[successIdx].end);
 
-        uint avgIdx = 6;
-        string memory avg = JsmnSolLib.getBytes(out, tokens[avgIdx].start, tokens[avgIdx].end);
+        uint twapIdx = 8;
+        string memory resultTWAP = JsmnSolLib.getBytes(out, tokens[twapIdx].start, tokens[twapIdx].end);
 
-        console.log("Did it Error:", isErr);
-        console.log("Average:", avg);
+        console.log("Success:", resultSuccess);
+        console.log("TWAP:", resultTWAP);
 
-        emit AttestedFunctionCallOutput(isErr, avg);
+        emit AttestedFunctionCallOutput(resultSuccess, resultTWAP);
```

Comment:
The changes here process the TWAP TA. As discussed, my plan is to connect with Lucid to see if this is not a problem for them. If its is, we can roll back to the Window TA